WooCommerce/Classes/Authentication/Jetpack Setup/Native Jetpack Setup/LoginJetpackSetupView.swift

```diff
@@ -0,0 +1,174 @@
+                                .resizable()
+                                .frame(width: Constants.stepImageSize * scale, height: Constants.stepImageSize * scale)
+                        }
+
+                        VStack(alignment: .leading, spacing: 4) {
+                            Text(step == .connection ? Localization.authorizing : step.title)
+                                .font(.body)
+                                .if(step <= viewModel.currentSetupStep) {
+                                    $0.bold()
+                                }
+                                .foregroundColor(Color(.text))
+                                .opacity(step <= viewModel.currentSetupStep ? 1 : 0.5)
+                            Label {
+                                Text(viewModel.currentConnectionStep.title)
+                                    .font(.footnote)
+                                    .fontWeight(.semibold)
+                            } icon: {
+                                viewModel.currentConnectionStep.imageName.map { name in
+                                    Image(systemName: name)
+                                }
+                            }
+                            .foregroundColor(Color(uiColor: viewModel.currentConnectionStep.tintColor))
+                            .renderedIf(step == .connection)
+                        }
+                    }
+                }
+                .padding(.top, Constants.contentVerticalSpacing)
+
+                Spacer()
+            }
+        }
+        .safeAreaInset(edge: .bottom, content: {
+            Button {
+                // TODO
+            } label: {
+                Text(Localization.goToStore)
+            }
+            .buttonStyle(PrimaryButtonStyle())
+            .padding(.top, Constants.contentVerticalSpacing)
+            .renderedIf(viewModel.currentSetupStep == .done)
+        })
+        .padding()
+    }
+}
+
+private extension LoginJetpackSetupView {
+    enum Localization {
+        static let installingJetpack = NSLocalizedString(
+            "Installing Jetpack",
+            comment: "Title for the Jetpack setup screen when installation is required"
+        )
+        static let connectingJetpack = NSLocalizedString(
+            "Installing Jetpack",
+            comment: "Title for the Jetpack setup screen when connection is required"
+        )
+        static let description = NSLocalizedString(
+            "Please wait while we connect your store %1$@ with Jetpack.",
+            comment: "Message on the Jetpack setup screen. The %1$@ is the site address."
+        )
+        static let goToStore = NSLocalizedString("Got to Store", comment: "Title for the button to navigate to the home screen after Jetpack setup completes")
+        static let authorizing = NSLocalizedString("Authorizing connection", comment: "Name of the connection step on the Jetpack setup screen")
```

Comment:
Super nit: Figma has a different spelling - "Authorising connection"

Reply:
I believe we want to use American English in the app (since A8C is an American company) - so I changed it intentionally.